pandas/core/series.py

```diff
@@ -223,7 +223,10 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
     name : str, optional
         The name to give to the Series.
     copy : bool, default False
-        Copy input data.
+        Copy input data. If False and the Series returns a `copy`
```

Comment:
Same comment on usage of "returns" as below. 

Generally I think this description might be too verbose for a parameter description and could be shortened to basically just get the idea across that a copy may still be made if necessary (as in the case of a list where we cannot share memory). Then the examples below should suffice to explain the different possible cases.

Reply:
(https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html has a good example of this)